fhir-persistence-jdbc/src/main/java/com/ibm/fhir/persistence/jdbc/dto/LocationParmVal.java

```diff
@@ -42,7 +45,16 @@ public void setValueLatitude(Double valueLatitude) {
     /**
      * We know our type, so we can call the correct method on the visitor
      */
+    @Override
     public void accept(ExtractedParameterValueVisitor visitor) throws FHIRPersistenceException {
         visitor.visit(this);
     }
+
+    @Override
+    public String getHash(ParameterHashUtil parameterHashUtil) {
+        StringBuilder sb = new StringBuilder();
```

Comment:
Same as previous comment. Compute hash from raw values instead of rendering as a string first.

Reply:
Updated to compute hash from raw values.